src/calculateRentalCost.js

```diff
@@ -3,8 +3,21 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
   // write code here
+  const dayCost = 40;
+  const discount3 = 20;
```

Comment:
The variable 'discount3' should be named to reflect the condition it is associated with, such as 'SHORT_TERM_DISCOUNT'.